src/Password/Bcrypt.php

```diff
@@ -19,7 +18,7 @@
  */
 class Bcrypt implements PasswordInterface
 {
-    const MIN_SALT_SIZE = 16;
+    const MIN_SALT_SIZE = 22;
```

Comment:
Can you please provide a link that details why the new size is preferred? (I'm curious!)